.ci/tidy.sh

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env sh
+
+# NOTE: just an adapted format.sh script
+
+# sample command line usage: $0 clang-tidy $SEISSOL_SOURCE_DIR $SEISSOL_BUILD_DIR [$ARGS for clang-tidy]
+# e.g. do, if SEISSOL_BUILD_DIR=$SEISSOL_SOURCE_DIR/build
+# $ cd $SEISSOL_SOURCE_DIR; sh ./.ci/tidy.sh ./ ./build
+# $ cd $SEISSOL_SOURCE_DIR/.ci; sh tidy.sh ../ ../build
+
+format() {
+    # don't use a directory with whitespace
+    local whitelist_dir="
+        src/DynamicRupture
+        "
+    
+    # NOTE: once the files of a directory are (almost) fully covered, consider moving it to whitelist_dir instead
+    local whitelist_file="
+        src/main.cpp
+        "
```

Comment:
Try to avoid white/blacklist and use allow/blocklist instead.

Reply:
Ok, changed it in the clang-format infrastructure as well